Dockerfile

```diff
@@ -450,6 +453,17 @@ RUN echo "BUILD_BRANCH=${BUILD_BRANCH}"    | tee /build.info \
     && echo "FDD_LENSES_CLI_VERSION=${LC_VERSION}" | tee -a /build.info \
     && sed -e 's/^/FDD_/' /opt/landoop/build.info  | tee -a /build.info
 
+# Add man pages & README.txt
+ADD ./doc/share/lenses /usr/local/share/lenses
+RUN apk add man-pages \
+      && for file in $(ls /usr/local/share/lenses/); do \
+      case ${file##*.} in \
+      '1'|'5') ln -sf "/usr/local/share/lenses/${file}" \
+      "/usr/share/man/man${file##*.}/${file}";; \
+      'txt') ln -sf "/usr/local/share/lenses/${file}" \
+      "/root/${file}"; ln -sf "/usr/local/share/lenses/${file}" \
+      "/${file}";; esac; done
```

Comment:
So, create `filesystem/usr/local/share/man/man1` and `filesystem/usr/local/share/man/man5`, add you man files there.

Now instead of doing this complex thing, just add to `filesystem/usr/local/share/landoop/etc/bashrc`:
```export MANPATH=/usr/share/man:/usr/local/share/man```